modules/serverless-kn-func-invoke.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-getting-started.adoc
+// * serverless/reference/kn-func-ref.adoc
+
+:_content-type: REFERENCE
+[id="serverless-kn-func-invoke_{context}"]
+= Invoking a deployed function with a test event
+
+You can use the `kn func invoke` CLI command to send a test request to invoke a function that is either deployed locally or deployed to your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly.
```

Comment:
You could minimise the first sentence in this line as follows, to enhance readability and assist with localisation:

> "You can use the `kn func invoke` CLI command to send a test request to invoke a function either locally or on your {product-title} cluster."